EIPS/eip-6372.md

```diff
@@ -64,24 +73,11 @@ The return string MUST be formatted like a URL query string (a.k.a. `application
       type: string
 ```
 
-### Solidity interface
-
-```sol
-interface IERC6372 {
-  function clock() external view returns (uint48);
-  function CLOCK_MODE() external view returns (string);
-}
-```
-
-### Expected properties
-
-- The `clock()` function MUST be non-decreasing.
```

Comment:
This is specified under Methods > clock.

Reply:
I would keep it. It's the most important property.